include/nds/arm9/video.h

```diff
@@ -54,8 +54,8 @@
 	tile and map offsets will be lost.
 
 	VRAM banks can be mapped to specific addresses for specific purposes.  In our
-	case, any of the 4 main banks amd several of the smaller ones can be mapped to
-	the main 2D background engine.(A B C and D banks are refered to as ìmainî
```

Comment:
Oh yea, just noticed the strange accents, just some garbled text, definitely not an identifier, will modify.